ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/agents.schema.yml

```
@@ -0,0 +1,34 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  agents:
+    type: list
+    primary_key: name
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+          description: Agent name.
+        environments:
```

Just a suggestion for clarity, but I can accept the one you have as well.
```suggestion
        environment_variables:
```

---

[6994314](https://github.com/aristanetworks/ansible-avd/pull/3282/commits/699431441517c470abe4c30ba61952e1bc6bd024)